### DISCUSSION TITLE
fix column title in program editor

### DISCUSSION THREAD

**Discussion by navlisilvan** [pretalx/pretalx#1437] [Ideas]
Hello,
It occurred to us that the program editor would be better to use if the column headings (the room names) are fixed when scrolling. 
Otherwise, for an event over many days with many rooms, you have to scroll up again and again to see where you are planning talks.

Thanks a lot for your work!

**Comment by rixx** [comment]
The schedule editor is currently being reworked entirely, see [here](https://twitter.com/pretalx/status/1611094943598104597).

**Reply by navlisilvan** [reply]
wow, amazing, thanks!

I don't even dare ask so as not to be rudely intrusive, but is there a rough timeline yet for when the new editor will be merged?

**Reply by rixx** [reply]
Nope. It's a matter of largely unpaid open source work, next to paid work, and other work on pretalx that has been commissioned/sponsored, so it takes precedence. The PR needs to include fully re-built translation capabilities for frontend units, a new build system, a new schedule editing API and much more. No way to rush it. You can follow progress at #1340 .
